packages/@aws-cdk/aws-cloudwatch/README.md

```diff
@@ -291,6 +291,75 @@ new cloudwatch.CompositeAlarm(this, 'MyAwesomeCompositeAlarm', {
 });
 ```
 
+## Contributor Insight Rules
+
+This construct library allows you to define Insight Rules.
+
+Contributor Insights Rules can be used to analyze log data to provide a view of top contributors.  Insight Rules take in key(s) in a log event to identify contributors for the analysis.  Log group(s) must be identified to specify which logs the the rule should search through for the analysis. The wildcard `*` can be appended to specify multiple log groups with a common prefix.
```

Comment:
Split this up into multiple lines, for better readability in the raw markdown. There's a couple more places to do this as well.